de.json

```
@@ -503,14 +503,14 @@
         "AvatarCreator.AlignToolAnchors" : "Werkzeug-Ankerpunkte ausrichten",
         "AvatarCreator.Create" : "Erstellen",
 
-        "Importer.Folder.Title" : "Ordner Importer",
+        "Importer.Folder.Title" : "Ordner Import",
```

I would suggest this as we used that for other things too, keeping the convention:
```suggestion
        "Importer.Folder.Title" : "Ordner-Import",
```